modules/ROOT/pages/rate-limiting.adoc

```diff
@@ -0,0 +1,237 @@
+= Rate Limiting Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: rate limiting, api gateway, gateway, policy
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | Rate Limiting
+>s| Summary      | Monitors access to an API by defining the maximum number of requests processed within a period of time
+>s| Category | Quality of Service
+>s| First Mule version available | v4.1.0
+.4+>.^s| Returned Status Codes
+|400 - Quota exceeded by WSDL APIs that use SOAP v1.2. Requests are blocked until the current window completes its processing.
+|429 - Quota exceeded, requests are blocked until the current window finishes
+|500 - Quota exceeded by WSDL APIs that use SOAP v1.1. Requests are blocked until the current window completes its processing.
+
+|===
+
+The Rate Limiting policy enables you to control the incoming traffic to an API by limiting the number of requests that the API can receive within a given period of time. After the limit is reached, the policy rejects all requests, thereby avoiding any additional load on the backend API.
+
+When you configure the Rate Limiting policy, you can specify any number of pairs of quota (number of requests) and time window (time period) values. 
+
+Additionally, you can further configure the policy to run in a Mule cluster, or add identifiers to define groups of requests. In a clustered policy configuration, the quota is shared among all of the nodes in the cluster. For more information about how these options work, see Examples.
+
+== How This Policy Works
+
+The Rate Limiting policy monitors the number of requests made in the current window (the available quota), allowing the requests to reach the backend only if the available quota is greater than zero. 
+
+You can configure the policy for multiple groups of requests by using identifiers in the policy configuration. Each group has a separate available quota for its window. 
+
+To understand how the Rate Limiting policy works, consider an example in which the configuration of 3 requests every 10 seconds allows or restricts incoming requests, based on the quota available in that window:
+
+image::rate-limiting-generic-example.png[Rate Limiting Policy Example,70%,85%]
+
+In the first window, because the quota is reached with the third request, all subsequent requests are rejected until the window closes. In the second window, only two of the three requests are processed and the quota remaining is dropped after the window time has elapsed.
+
+An accepted request passes through the API to the backend. Conversely, a rejected request displays 1429 status for HTTP (or either 400 or 500 if the API is WSDL) and does not reach the backend.
+
+A rejected request on the other hand, displays a “429 status for HTTP, ” (or either 400 or 500 if the API is WSDL) and does not reach the backend.
```

Comment:
I missed this typo in the gdoc, sorry! Could you change it for **“429 status for HTTP”,**

Reply:
Done! Thanks!